src/context/DataContext.jsx

```diff
@@ -0,0 +1,23 @@
+import { createContext, useContext, useState } from 'react';
+
+const DataContext = createContext();
+
+export const useData = () => useContext(DataContext);
+
+export const DataProvider = ({ children }) => {
```

Comment:
The name of the context is too generic, be more specific with names

Reply:
Added City- prefix!